packages/node/src/index.ts

```diff
@@ -85,14 +85,29 @@ export { getModuleFromFilename } from './module';
 export { enableAnrDetection } from './integrations/anr/legacy';
 
 import { Integrations as CoreIntegrations } from '@sentry/core';
+import type { Integration } from '@sentry/types';
 
 import * as Handlers from './handlers';
 import * as NodeIntegrations from './integrations';
 import * as TracingIntegrations from './tracing/integrations';
 
 const INTEGRATIONS = {
   ...CoreIntegrations,
-  ...NodeIntegrations,
+  ...(NodeIntegrations as {
+    Console: Integration;
+    Http: typeof NodeIntegrations.Http;
+    OnUncaughtException: Integration;
+    OnUnhandledRejection: Integration;
+    Modules: Integration;
+    ContextLines: Integration;
+    Context: Integration;
+    RequestData: Integration;
+    LocalVariables: Integration;
+    Undici: typeof NodeIntegrations.Undici;
+    Spotlight: Integration;
+    Anr: Integration;
+    Hapi: Integration;
```

Comment:
why do we need this cast?

Reply:
nvm just saw your other comment, let's figure this out later. Would leave a comment about why we need to cast though!